### DISCUSSION TITLE
How can I filter a histogram dashboard panel for traces?

### DISCUSSION THREAD

**Discussion by jansorg** [SigNoz/signoz#5190] [Q&A]
I'd like to create a histogram dashboard panel for `durationNano` for a certain trace event.
It seems that the query field next to `Traces` is defining what's used for the buckets. Now I don't know how to filter and define the value for the buckets at the same time.

I'd like to show `durationNano` in the histogram, but only for trace events with `name = shellcheck.execute`.

This doesn't work, it's always using `1` on the Y axis:
![image](https://github.com/SigNoz/signoz/assets/11473063/8577acdf-e347-4f3a-acd2-dbac9cf26a9d)

Thanks!

**Comment by srikanthccv** [comment]
>This doesn't work, it's always using 1 on the Y axis:

What were you expecting it to show? You are summing the `durationNano` for an interval. The result of the aggregation is one `SUM(durationNano)` for each interval. This result then gets plotted as histogram.
